### ISSUE TITLE
Implicit labels for checkboxes are not read if there is additional whitespace in the code

### ISSUE THREAD

**Issue by victorloux** [FreedomScientific/VFO-standards-support#115]
### Summary

When using an implicit label and the `<input>` is the *last element* in this label, JAWS does not announce the label when tabbing on the input if there is whitespace (except new lines) in HTML between the `<input>` and the `</label>`.

For example this is announced as expected:

```html
<label>
    Buying equipment
    <input type="checkbox" checked="checked">
</label>
```

This isn't:

```html
<label>
    Buying equipment
    <input type="checkbox" checked="checked">
    </label>
```

Note the four spaces before the `</label>` here; this is the only difference. (I have added four for clarity but this happens even with one space on that line)

The label is announced as expected when arrowing down with IE, but with Edge + JAWS2018 it is not announced at all even when arrowing.
This issue happens on radio buttons and checkboxes, but not text fields.

### Expected result

When tabbing onto the checkbox, the name. role and value should be announced ("Buying equipment, checkbox not checked").

### Actual result

Only the role and value are announced ("Checkbox not checked"), but not the label.

(On IE the label was read when arrowing down, but not with Edge.)

### Example

https://codepen.io/vloux/pen/dqwGpQ

(debug link for IE: https://s.codepen.io/vloux/debug/dqwGpQ)

### Additional Information

This happens even if the implicit label is redundantly made explicit (with the `for` attribute). This was the case on the client's website where I found the issue.

Accessibility Viewer displays the label, so it seems to be read fine by IE but lost along the way in the API or in the way JAWS processes the label. However this works as expected with Firefox so unsure if this is an issue with IE/Edge only.

#### JAWS version and build number

JAWS 2018 1080.1808.10
JAWS 18.0.4321
 
#### Operating System and version

Windows 10 Pro build 1803

#### Browser and version:

IE 11.285.17134.0
Edge

**Comment by lamador** [comment]
The implicit checkbox label of "Fixing Equipment" in the provided code sample was read when using the latest versions of JAWS 2020 (ver 2020.2004.66), IE11 (ver 11 11.778.18362.0) and Edge (ver 44.18362.449.0).  The exception was when using the Tab key in IE11 to navigate to the checkbox when the heading "Works (label text after the input)" following the checkbox is announced instead.
![Screenshot of heading text announced as checkbox label instead of implicit "Fix Equipment" label](https://user-images.githubusercontent.com/332233/81754696-cf231c00-9484-11ea-9b67-225937bdefe1.png)

**Comment by TPGJRogers** [comment]
Issue is marked IE11. Closing.
